udata/core/site/views.py

```diff
@@ -169,13 +170,6 @@ def get_context(self):
                     'endpoint': 'organizations.list',
                 },
                 {
```

Comment:
Not very much, but it's something that is used by the team to follow a little bit our "lobbying" progression.
Keep it like, we will see if someone ask for this.